code/game/gamemodes/umbrage/umbrage_minor_abilities.dm

```diff
@@ -45,23 +45,25 @@
 	var/victims = list() //A list of people we've used the bead on recently; we can't drain them again so soon
 
 /datum/action/innate/umbrage/devour_will/IsAvailable()
-	if(!usr || !usr.get_empty_held_indexes())
+	var/mob/living/user = usr
```

Comment:
>not knowing how to use the system before using it

Damnit Xhuis action datums are one of our better put together frameworks, they're super simple to get your head around!